index.html

```diff
@@ -232,111 +232,111 @@ <h1>Connect with Fossasia on Social Media</h1>
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/2.webp" data-lightbox="Gallery">
+          <img id="memory2" class="memories_img img-fluid" src="img/PastMemories/2.webp">
+        </a>
+      </div>
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/3.webp" data-lightbox="Gallery">
+          <img id="memory3" class="memories_img img-fluid" src="img/PastMemories/3.webp">
+        </a>
+      </div>
+      <div class="col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/4.webp" data-lightbox="Gallery">
+          <img id="memory4" class="memories_img img-fluid" src="img/PastMemories/4.webp">
+        </a>
+      </div>
+      <div class="col-md-2 spacing-0">
+        <a href="img/PastMemories/5.webp" data-lightbox="Gallery">
+          <img id="memory5" class="memories_img img-fluid" src="img/PastMemories/5.webp">
+        </a>
+      </div>
+      <div class="col-md-2 spacing-0">
+        <a href="img/PastMemories/6.webp" data-lightbox="Gallery">
+          <img id="memory6" class="memories_img img-fluid" src="img/PastMemories/6.webp">
+        </a>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/7.webp" data-lightbox="Gallery">
+          <img id="memory7" class="memories_img img-fluid" src="img/PastMemories/7.webp">
+        </a>
+      </div>
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/8.webp" data-lightbox="Gallery">
+          <img id="memory8" class="memories_img" src="img/PastMemories/8.webp">
+        </a>
+      </div>
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/9.webp" data-lightbox="Gallery">
+          <img id="memory9" class="memories_img" src="img/PastMemories/9.webp">
+        </a>
+      </div>
+      <div class="col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/10.webp" data-lightbox="Gallery">
+          <img id="memory10" class="memories_img" src="img/PastMemories/10.webp">
+        </a>
+      </div>
+      <div class="col-md-2 spacing-0">
+        <a href="img/PastMemories/11.webp" data-lightbox="Gallery">
+          <img id="memory11" class="memories_img" src="img/PastMemories/11.webp">
+        </a>
+      </div>
+      <div class="col-md-2 spacing-0">
+        <a href="img/PastMemories/12.webp" data-lightbox="Gallery">
+          <img id="memory12" class="memories_img" src="img/PastMemories/12.webp">
+        </a>
+      </div>
     </div>
+    <div class="row">
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/13.webp" data-lightbox="Gallery">
+          <img id="memory13" class="memories_img" src="img/PastMemories/13.jpg">
+        </a>
+      </div>
+      <div class="col-xs-4 col-sm-3 col-md-2 spacing-0">
+        <a href="img/PastMemories/14.jpg" data-lightbox="Gallery">
```

Comment:
This is not converted, please look carefully. Convert Till 18

Reply:
Alright..